src/lib/components/forms/form/form-components/slider/slider.component.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { SliderComponent } from './slider.component';
+import { HarnessLoader, TestElement } from '@angular/cdk/testing';
+import { MatSliderHarness } from '@angular/material/slider/testing';
+import { MatSlider, MatSliderModule } from '@angular/material/slider';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule } from '@angular/forms';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatFormFieldHarness } from '@angular/material/form-field/testing';
+import { By } from '@angular/platform-browser';
+import { TerraPlacementEnum } from '../../../../../helpers';
+
+describe('SliderComponent', () => {
+    let fixture: ComponentFixture<SliderComponent>;
+    let component: SliderComponent;
+    let loader: HarnessLoader;
+    let slider: MatSliderHarness;
+
+    beforeEach(async () => {
+        TestBed.configureTestingModule({
+            imports: [MatSliderModule, NoopAnimationsModule, FormsModule],
+            declarations: [SliderComponent, MockTooltipDirective]
+        });
+
+        fixture = TestBed.createComponent(SliderComponent);
+        component = fixture.componentInstance;
+        loader = TestbedHarnessEnvironment.loader(fixture);
+
+        fixture.detectChanges();
+
+        slider = await loader.getHarness(MatSliderHarness);
+    });
+
+    it('should create', async () => {
+        expect(component).toBeTruthy();
+        expect(slider).toBeTruthy();
+    });
+
+    it('should disable the input when #isDisabled is set', async () => {
+        expect(await slider.isDisabled()).toBe(false);
+        component.isDisabled = true;
+        fixture.detectChanges();
+
+        expect(await slider.isDisabled()).toBe(true);
+    });
+
+    it('should have #name as label of the input', async () => {
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        expect(await formField.getLabel()).toBe('');
+
+        component.name = 'My Label';
+        fixture.detectChanges();
+        expect(await formField.getLabel()).toBe(component.name);
+    });
```

Comment:
Hmm.. Ich glaube der slider kann nicht in ein MatFormField eingebettet werden. Das mit dem Label wird also schwer.

Reply:
done